src/CHG.Extensions.Security.Txt.csproj

```diff
@@ -1,42 +1,48 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Company>CHG-MERIDIAN AG</Company>
-    <Authors>Toni Wenzel</Authors>
-    <Description>Provides a middleware for the security.txt (see https://securitytxt.org/)</Description>
-    <Version>$(SemVer)</Version>
-    <PackageTags>security securitytxt middleware security.txt</PackageTags>
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <PackageIconUrl>https://www.gravatar.com/avatar/5ebe9d32332f00c9c3496fe193e863f8?d=package.png</PackageIconUrl>
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
Why do we need `netcoreapp3.1`? Isn't `netstandard2.0` already covering that?
![image](https://user-images.githubusercontent.com/2471424/154213342-745fd710-9e6b-4fad-9328-0338c3a5d595.png)

Reply:
To get the "correct" dependencies
![image](https://user-images.githubusercontent.com/500376/154216767-68a42a86-69e0-4887-ab0c-1fa617d175a8.png)